flink-connectors/flink-connector-jdbc/src/test/java/org/apache/flink/connector/jdbc/table/JdbcTableSourceSinkFactoryTest.java

```diff
@@ -255,6 +255,17 @@ public void testJdbcValidation() {
 			fail("exception expected");
 		} catch (IllegalArgumentException ignored) {
 		}
+
+		// lookup max-retries properties not complete
+		try {
+			Map<String, String> properties = getBasicProperties();
+			properties.put("connector.lookup.max-retries", "0");
```

Comment:
Zero is a valid value, so there should no exception be thrown here. I think what we want to test is -1 here.